system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -68,6 +68,38 @@ LDAP* LdapUtils::LdapInit(const char* protocol, const char* host, int port, int
 #ifndef HAVE_TLS
         //throw MakeStringException(-1, "openldap client library libldap not compiled with TLS support");
 #endif
+
```

Comment:
I think all this should be done both if ldaps or just ldap, so not in this if (stricmp(ldaps)) block but before or after.  And probably also ok if _WIN32.